server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -130,19 +130,15 @@ public void unregister(CommandHandler handler) {
     /**
      * Verifies if the command can be posted to this {@code CommandBus}.
```

Comment:
I think we need to say that the method verifies a command message, not a command.